src/host/main.cpp

```diff
@@ -130,7 +146,17 @@ int main(int argc, char** argv)
     return 0;
   }
 
-  LOG_INFO_FMT("Configuration file {}:\n{}", config_file_path, config_str);
+  LOG_INFO_FMT(
+    "Configuration file {}:\n{}", config_file_path, config_json.dump(2));
+
+  nlohmann::json environment;
+  for (int i = 0; environ[i] != nullptr; i++)
+  {
+    auto [k, v] = nonstd::split_1(environ[i], "=");
+    environment[k] = v;
+  }
+
+  LOG_INFO_FMT("Environment: {}\n", environment.dump(2));
```

Comment:
Note: we now also dump (formatted as a nice JSON object, on a single line) the environment variables passed to the node on startup.